apps/hash-frontend/src/components/block-loader/block-loader.tsx

```diff
@@ -216,15 +211,16 @@ export const BlockLoader: FunctionComponent<BlockLoaderProps> = ({
     }
 
     /**
-     * Our text blocks ask for a BP `textual-content` property, which is plain `Text`.
-     * Because they use the hook service, we actually generate text as text tokens and persist it under a different property.
-     * The BP spec says that blocks should receive the data they expect even if the hook service is used
-     * – this code makes sure we provide rich text as a plain string for this specific property.
+     * Our text blocks ask for a BP `textual-content` property as a plan `string`ich is plain `Text`.
+     * Because they use the hook service, we actually generate text as an array of text tokens and
+     * persist it under a different property.
+     * The BP spec says that blocks should receive the data they expect even if the hook service is used,
+     * therefore we provides the rich text as a plain string for the `textual-content` property.
```

Comment:
```suggestion
     * Text fields use a `textual-content` property, with a value of either `Text` (a string) or an opaque `Object`.
     * This is so that the text can be stored as rich text tokens by the embedding application, when the hook service is used,
     * but also still receive by the block as a plain string (a predictable format), complying with the schema in both cases.
     * Here we translate our rich text tokens into a plain string for passing into the block.
```

Reply:
Thanks for spotting, have updated it in https://github.com/hashintel/hash/pull/3522/commits/fc8a7a64ddbc3351b4c04b02c5ea4b3dfb4ff13e